dist/src/main/amoro-bin/conf/config.yaml

```diff
@@ -54,6 +54,7 @@ ams:
     task-ack-timeout: 30000 # 30s
     polling-timeout: 3000 # 3s
     max-planning-parallelism: 1 # default 1
+    ignore-filter-partition-count: 100 # default 100
```

Comment:
I think this param is not suitable for this place.
It's more suitable for self-optimizing groups.

And I think more proper meaning is 'self-optimizing.skip-evaluating-for-partition-count', WDYT @zhoujinsong

Reply:
I think it's appropriate to put it in self-optimizing groups.

But I think `skip-evaluating` is inappropriate, because this parameter only controls whether to use the filter, not skipping the Evaluator.

Reply:
Yes, that's correct. I think skip is inappropriate too.

In original there are no evaluating and filtering. But it is proved that directly planning would cause OOM, because all partition files would be cached in memory, which leads to a evaluating phase before planning(stream planning and store a partition set to avoid memory usage in planning phase).  If we do not filter here, that means evaluating is not used or skipped. 

Evaluating is a sensible concept and even be revealed on dashboard in the future, however filter is quite a detailed implementation and could be pointed to anything evolved.

How do you think using a set to filter to resolve the issue directly?